universal-login-react/src/ui/Notifications/ConnectionNotification.tsx

```diff
@@ -42,7 +37,7 @@ export const ConnectionNotification = ({deployedWallet, devicesBasePath, classNa
                 deployedWallet={deployedWallet}
                 hideTitle={() => setShowTitle(false)}
                 className={className}
-                onDenyRequests={onDenyButtonClick}
+                onDenyRequests={() => history.goBack()}
```

Comment:
Since it does not deny requests anymore, shall we rename it from `onDenyRequests` to `onCancel` or something?

Reply:
It is denyRequests but in EmojiForm

Reply:
`onRequestsDenied`

Reply:
✔️